about.html

```diff
@@ -12,62 +12,99 @@
 </head>
 
 <body>
+      
+    <div class="static-navbar">
+        <nav>
+            <div>
+             <img class="logo" src="C:\Users\Owner\Desktop\Lambda-Git\Sprint-Challenge--User-Interface\img\lambda-black.png">
+             </div> 
+            <span>
+             <a href="index.html">Home</a>
+             <a href="about.html">About</a>
+             <a href="#">Products</a>
+             <a href="#">Blog</a>
+             <a href="#">Contact</a>
+            </span> 
+         
+        </nav>
+    </div>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
+        <div class="road">
+            <img class="road-m" src=C:\Users\Owner\Desktop\Lambda-Git\Sprint-Challenge--User-Interface\img\jumbo-about.png>
+        </div>    
+    <section class="header">
+      <h1>About
```

Comment:
throwing this item in a div would help control its with on the page. If you wanted to give it a 90% it would need a parent div